Tools-Override/tests.targets

```diff
@@ -130,6 +130,10 @@
          when the properties are set currently. -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+
+      <!-- We also need to replace {CoverageFilter} with its true value, which should be evaluated correctly at this point. -->
+      <TestCommandLine>$(TestCommandLine.Replace('{CoverageFilter}', '$(CoverageFilter)'))</TestCommandLine>
```

Comment:
Interesting.  I discussed with @danmosemsft and he mentioned it sounded like a parsing bug.  Regardless, this seems ok.